packages/client/wallets/aa/src/utils/log.ts

```diff
@@ -98,7 +98,7 @@ export function logInputOutput(fn: Function, functionName: string) {
                     .catch((err) => {
                         logError(`${identifierTag} threw_error: ${beautify(err)}`, { err });
                         throw err;
-                    });
+                    }) as unknown as T;
```

Comment:
Not %100 sure on this change:
![image](https://github.com/Crossmint/crossmint-sdk/assets/7073959/7481d0fe-6d1f-49ad-a8f1-2aba682372fc)
Not sure it would work on async calls, haven't really tested it. 
We may need an `async logInputOutput` ? For the backlog backlog

Reply:
https://crossmint.slack.com/archives/C07078U8Y04/p1717004537259129?thread_ts=1716991730.255829&cid=C07078U8Y04